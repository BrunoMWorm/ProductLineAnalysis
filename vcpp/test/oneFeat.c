int x=0;
#ifdef A
int y=1;
#endif

void main()
{
}
