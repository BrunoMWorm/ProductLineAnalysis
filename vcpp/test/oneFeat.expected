#A#
int x=0;

int y=1;


void main()
{
}

#Not(A)#
int x=0;




void main()
{
}